### PR TITLE
docs: add docs about defaultIgnore config

### PR DESCRIPTION
**Pull request by robin-drexler**
Docs for https://github.com/testing-library/dom-testing-library/pull/1138

### CODE REVIEW

File: docs/dom-testing-library/api-configuration.mdx [hunk 1]
```diff
@@ -80,6 +80,11 @@ with a second argument e.g.
 The default value for the `hidden` option used by
 [`getByRole`](queries/byrole.mdx). Defaults to `false`.
 
+### `defaultIgnore`
+
+The default value for the [`ignore` option](queries/bytext.mdx#ignore) used by
+[`getByText`](queries/bytext.mdx). Defaults to `script, style`.
```
Comment (eps1lon) [inline docs/dom-testing-library/api-configuration.mdx:86]: Now that I think about it: It's not just `*ByText`.This also affects the nodes we print in error messages.

**Review by eps1lon — CHANGES_REQUESTED** [review]

File: docs/dom-testing-library/api-configuration.mdx (hunk 1 above)
Comment (robin-drexler) [reply docs/dom-testing-library/api-configuration.mdx:86]: @eps1lon do you have a specific wording you'd like me to put here? 👀

Comment (robin-drexler) [reply docs/dom-testing-library/api-configuration.mdx]: @eps1lon I gave it a shot. 😅 What do you think?

**Review by eps1lon — APPROVED** [review]

**Comment by eps1lon** [comment]
@all-contributors add @robin-drexler for docs

**Pull request merged**